pylint/extensions/typing.py

```diff
@@ -124,10 +124,10 @@ class TypingChecker(BaseChecker):
                 "type": "yn",
                 "metavar": "<y_or_n>",
                 "help": (
-                    "Set to ``no`` if the app / libary does NOT need to "
-                    "support runtime introspection of type "
-                    "annotations. Only applies to Python version "
-                    "3.7 - 3.9"
+                    "Set to ``no`` if the app / library does NOT need to "
+                    "support runtime introspection of type annotations. "
+                    "Only change it if you understand what that means. "
```

Comment:
```suggestion
                    "Only change it if you understand what that means. "
```
Could we explain "what that means" or link to the doc that explains it here ? :)

Reply:
My initial idea was something along the lines:
```
If you don't know what that means, you're probably better off to leave it as is.
```

I just wasn't sure this would be the right tone. It's never the less true. This should really only be set if the user fully understands the consequences. Not knowing `runtime introspection` is a clear signal that he/she doesn't know enough.

Reply:
Sure, but this is the place where we should link to where you need to look if you want to try to understand, no ?

Reply:
@Pierre-Sassoulas I've updated the explanation. Please take a look and let me know what you think